java-client/pom.xml

```diff
@@ -317,20 +317,21 @@
           </execution>
         </executions>
       </plugin>
+      <!--
+      TODO(yingchun): we have to fix the issues reported by spotbugs
```

Comment:
I added an issue to track it https://github.com/apache/incubator-pegasus/issues/1144